examples/.scripts/run-example.sh

```diff
@@ -10,12 +10,50 @@
 
 result () {
     local cmd="$1"
-    if ($cmd &> /dev/null)
+    local args="$2"
+    local file="$3"
+
+    # Skip verification for specs that do not define a state machine
```

Comment:
@shonfeder I know you will object to the added complexity here, but all of this complexity will go away once we fix the relevant issues 😃

Reply:
If it helps you and I don't have to maintain it, who am I to object :wink:x